packages/amplify-graphql-model-transformer/src/resources/rds-model-resource-generator.ts

```diff
@@ -1,12 +1,23 @@
+import { MYSQL_DB_TYPE, RDSConnectionSecrets } from '@aws-amplify/graphql-transformer-core';
```

Comment:
I feel like we're overloading the model transformer with all this functionality. Is there a way to do this outside of this transformer? Since patching is not exactly a concern of a GraphQL transformer.

Reply:
Model directive is mostly responsible for provisioning the datasource. In this case, patching the lambda layer and SQL lambda are core components of datasource for RDS models. The lambda code is not big enough to move into a new package yet. We can start with having them here and move it easily if needed.